app/src/main/java/org/wikipedia/bridge/CommunicationBridge.java

```diff
@@ -49,6 +49,7 @@ public CommunicationBridge(final WebView webView) {
         webView.getSettings().setAllowUniversalAccessFromFileURLs(true);
         webView.getSettings().setMediaPlaybackRequiresUserGesture(false);
         webView.setWebChromeClient(new CommunicatingChrome());
+        webView.addJavascriptInterface(new CommunicatingChrome(), "pcsClient");
```

Comment:
Why are we creating another instance of `CommunicatingChrome` here? That's not what that class is intended for.  Couldn't we just add the new setup method to `BridgeMarshaller` and rename it from `marshaller` to `pcsClient`?